common/lib/dependabot/clients/azure.rb

```diff
@@ -310,6 +317,13 @@ def truncate_pr_description(pr_description)
         pr_description.force_encoding(Encoding::UTF_8)
       end
 
+      def tags_creation_forbidden?(response)
+        return if response.body.empty?
+
+        message = JSON.parse(response.body).fetch("message", nil)
+        message&.include?("TF401289: The current user does not have permissions to create tags")
```

Comment:
Do you think it would be sufficient to just match on `TF401289`, that way if the copy in that error ever changes we don't need to patch this code, and it seems like this is a code that specifically represents this tags permission error?

Reply:
Seems like that works, as `TF401289` is the message code for the permission tags issue. Will update the fix